packages/core/android/src/main/kotlin/com/datadog/reactnative/JSONFileReader.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.reactnative
+
+import android.content.Context
+import java.io.BufferedReader
+import java.io.InputStreamReader
+
+/**
+ * Utility for parsing JSON files from the app assets.
+ */
+class JSONFileReader {
+    internal fun parseAssetsJSONFile(appContext: Context, filePath: String): String {
+        val file = appContext.assets.open(filePath)
+        val bufferedReader = BufferedReader(InputStreamReader(file))
+
+        val stringBuilder = StringBuilder()
+        bufferedReader.useLines { lines ->
+            lines.forEach {
+                stringBuilder.append(it)
+            }
+        }
+        
+        return stringBuilder.toString()
```

Comment:
it can be simplified as:

```suggestion
        return appContext.assets.open(filePath).use {
          it.bufferedReader().readText()
        }
```